ArduPlane/fence.cpp

```diff
@@ -0,0 +1,132 @@
+#include "Plane.h"
+
+// Code to integrate AC_Fence library with main ArduPlane code
+
+#if AC_FENCE == ENABLED
+
+// fence_check - ask fence library to check for breaches and initiate the response
+// called at 1Hz
```

Comment:
No it's not :-P

These comments always get out of date.  Just remove this "called at" bit.